src/wavelet.cpp

```diff
@@ -1342,18 +1344,23 @@ Wavelet::WaveletReadJason(const std::string & fileName,
   int cz_file = static_cast<int>(floor((fabs(shift/dz_))+0.5));
   cz          = nz_ - cz_file; //Wavelets are flipped
 
+  rAmp[0] = 0.0f;
   for(int i=0; i<nz_;i++) {
     if (NRLib::CheckEndOfFile(file)) {
       errText += "Error: End of file "+fileName+" premature.\n";
       errCode=1;
       return;
     }
     NRLib::ReadNextToken(file,dummyStr,line);
-
-    rAmp[nz_-i-1] = NRLib::ParseType<float>(dummyStr); //Flip wavelet
+    //
+    // The wavelet fill in starts at i=1 and rAmp[0] is set to zero.
+    //
+    // If the fill in starts at 0, a wavelet read from file gets an incorrect
+    // shift of one sample. This is particular evident if a Ricker wavelet
+    // read from file is compared the the wavelet given as output.
+    //
+    rAmp[nz_ - i] = NRLib::ParseType<float>(dummyStr); //Flip wavelet
```

Comment:
Denne er relatert til "float shift = -dznew*halfLength;" (linje 645) for skriving av wavelet?
Burde ikke WaveletReadNorsar også få samme fiksen?